sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/models/FaceAttributeType.java

```diff
@@ -181,17 +180,21 @@ public static final class Detection01 {
      */
     public static final class Detection03 {
 
+        /**
+         * Face is blurry or not. Level returns 'Low', 'Medium' or 'High'. Value returns a number between [0,1], the
+         * larger the blurrier.
+         */
+        public static final FaceAttributeType BLUR = FaceAttributeType.BLUR;
+
         /**
          * 3-D roll/yaw/pitch angles for face direction.
          */
-        @Generated
```

Comment:
why is this?

Reply:
We actually support this attributes for detection03, but our documentation wasn't updated. So I missed this attributes. We discover it in recent documentation migration.

Reply:
Ah, all this is added by hand?

Reply:
Yes.